src/dodal/devices/slits/slit_motor.py

```diff
@@ -0,0 +1,72 @@
+from collections import OrderedDict
+from enum import IntEnum
+
+from bluesky.protocols import Movable, Readable
+from ophyd import Component, Device, EpicsSignal, EpicsSignalRO
+from ophyd.status import SubscriptionStatus
+
+from dodal.devices.status import await_value
+
+
+class MoveState(IntEnum):
+    Moving = (0,)
+    Stationary = 1
+
+
+class SlitMotor(Device, Movable, Readable):
+    """A class representing a slit motor.
```

Comment:
Nit: Sorry this is really nitpicky, but the first line of this comment tells us the same thing as the class definition line